app/views/cloud_topology/show.html.haml

```diff
@@ -0,0 +1,51 @@
+- tooltipOptions = {"tooltip-placement" => "bottom", "tooltip" => "{{legendTooltip}}"}
+.container_topology{'ng-controller' => "cloudTopologyController"}
+  .row.toolbar-pf
+    .col-md-12
+      = render :partial => "shared/topology_header"
+  .legend
+    %label#selected
+    %div{'ng-if' => "kinds"}
+      %kubernetes-topology-icon{tooltipOptions, :kind => "Vm"}
```

Comment:
ok, sounds good to defer. Talk to @himdel then, what would be the best way to extract and share this code